ADOPTERS.md

```diff
@@ -8,5 +8,5 @@ name to this list (in alphabetical order).
 
 | Organization                       | Contact                                          | How you use the AEPs                                                                                                                                                                                                                           |
 | ---------------------------------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [Google](https://about.google/)    | [@toumorokoshi](https://github.com/toumorokoshi) | Google maintains a fork at [google.aep.dev](http://google.aep.dev) which is adhered to for all APIs. Google also runs the [api-linter](https://linter.aep.dev/) across the company internally.                                                 |
-| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AEPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aep.dev) and [site generator](https://github.com/aep-dev/site-generator). |
+| [Google](https://about.google/)    | [@toumorokoshi](https://github.com/toumorokoshi) | Google maintains a fork at [google.aip.dev](http://google.aip.dev) which is adhered to for all APIs. Google also runs the [api-linter](https://linter.aip.dev/) across the company internally.                                                 |
+| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AEPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aip.dev) and [site generator](https://github.com/aep-dev/site-generator). |
```

Comment:
```suggestion
| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AEPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aip.dev) and [site generator](https://github.com/aep-dev/site-generator). |
```

Actually Google is not affiliated with the AEPs (at least not officially) - we can remove this line for now.

Reply:
I removed the section entirely.